### DISCUSSION TITLE
Game not showing up

### DISCUSSION THREAD

**Discussion by lca211** [OnionUI/Onion#1715] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3.0

### Miyoo Firmware Version

my354

### Provide a clear and concise description of the issue

 downloaded a game, but it doesn't show up in my library. I can only open the game with RetroArch, and it works that way. The game in question is Pokémon Unbound.

I would like it to appear in the regular library so I can pause and manage it like the other games.

### Steps to Reproduce

Download the game 

### Provide any additional context or information that may help us investigate your issue

---

**Comment by Sichroteph** [comment]
Try to refresh your rom list. 
(Press select in your consoles view)

**Reply by lca211** [reply]
does nothing…

**Reply by Sichroteph** [reply]
The select button does nothing, or the refresh does nothing?
If it is the refresh, check if the game file is zipped and try to unzip it in the GBA folder

**Reply by lca211** [reply]
I refreshed it, but it doesn’t show up. It’s a normal .gba file. I have similar ones on my device, and they work.

**Comment by Sichroteph** [comment]
Strange. If the extension is .gba and the file is in your correct rom folder, it should show up after a refresh.

**Comment by thomasklemenc** [comment]
Have you scrolled all the way down on your lists? Sometimes the sorting can be messed because of extra characters etc.
